components/aws/sagemaker/tests/integration_tests/conftest.py

```diff
@@ -86,12 +87,24 @@ def kfp_client():
     kfp_installed_namespace = utils.get_kfp_namespace()
     return kfp.Client(namespace=kfp_installed_namespace)
 
+def get_experiment_id(kfp_client):
+    exp_name = datetime.now().strftime("%Y-%m-%d-%H-%M")
+    experiment = kfp_client.create_experiment(name=exp_name)
+    return experiment.id
 
 @pytest.fixture(scope="session")
-def experiment_id(kfp_client):
-    exp_name = datetime.now().strftime("%Y-%m-%d")
-    try:
-        experiment = kfp_client.get_experiment(experiment_name=exp_name)
-    except ValueError:
-        experiment = kfp_client.create_experiment(name=exp_name)
-    return experiment.id
+def experiment_id(kfp_client, tmp_path_factory, worker_id):
+    if not worker_id:
```

Comment:
Nice, please add the source link https://github.com/pytest-dev/pytest-xdist#making-session-scoped-fixtures-execute-only-once